pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -490,6 +513,14 @@ type FrameworkHandle interface {
 	ClientSet() clientset.Interface
 
 	SharedInformerFactory() informers.SharedInformerFactory
+
+	// TODO: unroll the wrapped interfaces to PreemptHandle.
+	PreemptHandle() PreemptHandle
```

Comment:
why add it in this PR at all?

Reply:
I thought you meant to put the unrolling to another PR instead of introducing PreemptHandle(). The integration test needs it.

Reply:
ok, I didn't notice it is being used in the integration test.